<ui version="4.0" >
 <class>lookandfeel</class>
 <widget class="QWidget" name="lookandfeel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QComboBox" name="styleSelector" />
     </item>
     <item>
      <widget class="QGroupBox" name="previewBox" >
       <property name="title" >
        <string>Preview</string>
       </property>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QGroupBox" name="previewGroupBox" >
            <property name="title" >
             <string>GroupBox</string>
            </property>
            <layout class="QVBoxLayout" >
             <property name="margin" >
              <number>9</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <layout class="QVBoxLayout" >
               <property name="margin" >
                <number>0</number>
               </property>
               <property name="spacing" >
                <number>6</number>
               </property>
               <item>
                <widget class="QRadioButton" name="previewRadio_1" >
                 <property name="text" >
                  <string>RadioButton</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="previewRadio_2" >
                 <property name="text" >
                  <string>RadioButton</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="previewLine" >
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="previewCheckBox" >
                 <property name="text" >
                  <string>CheckBox</string>
                 </property>
                 <property name="checked" >
                  <bool>true</bool>
                 </property>
                 <property name="tristate" >
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QSlider" name="previewSlider" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QSpinBox" name="previewSpinBox" />
              </item>
              <item>
               <widget class="QPushButton" name="previewPushButton" >
                <property name="text" >
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QComboBox" name="previewComboBox" />
            </item>
           </layout>
          </item>
          <item>
           <widget class="QScrollBar" name="previewScrollBar" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
